### DISCUSSION TITLE
Unable to use comma separated parameter value in DataunctionHandler

### DISCUSSION THREAD

**Discussion by gourighode** [pactumjs/pactum#336] [Q&A] [answered]
We are using DataFunctionHandler concept to prepare request body with dynamic values for some keys.

In some of the scenarios value of specific key is comma separated 

Example of Sample handler

```js
handler.addDataFuncHandler(’SamplePayLoad', (ctx) => {
  const source = ctx.args[0];
  console.log("*********************",source)
  const basePayload = {
    source: ctx.args[4],
  };
  return basePayload;
}); 
```

Value being passed for source while calling data function handler in actual test is A,B this combined value is needed for source however currently it is splitting it between 2 different parameters and hence value for source is appearing as only A

Can someone help in clarifying if is there any way to tackle this issue?

**Comment by leelaprasadv** [comment, marked as answer]
Replied in
https://github.com/pactumjs/pactum/issues/337#issuecomment-2035222531
